signalfx-tracing/signalfx-java-tracing/jetty-customtags/README.md

```diff
@@ -0,0 +1,25 @@
+# Java Custom Tag Example Using Jetty
+
+This example assumes an already configured SignalFx SmartGateway and SmartAgent. They can be running on the same machine as the example below however it is recommended to have at least 6GB RAM for this test.
```

Comment:
Given the content I wouldn't stipulate the memory requirement.

Reply:
I noticed that unless >4GB of RAM was present while testing this on the same machine as a SmartGateway that the SmartGateway was running out of memory and crashing hence my suggestion for memory of that size.

Reply:
Since this is public facing I think deferring to official sizing requirements would be ideal rather than specifying something that could be used as a reference https://docs.signalfx.com/en/latest/apm/apm-deployment/smart-gateway.html#sizing-recommendations.